dive_core/command_hierarchy.cpp

```diff
@@ -1801,13 +1803,28 @@ void CommandHierarchyCreator::AppendRegNodes(const IMemoryManager &mem_manager,
         if (reg_info_ptr == nullptr)
             reg_info_ptr = &temp;
 
-        DIVE_ASSERT(!reg_info_ptr->m_is_64_bit);
+        uint64_t reg_value = reg_pair.m_reg_value;
```

Comment:
Sorry i am a bit confused here, could you add some comments about the layout for both 32bit and 64bit?

Is this understanding correct?
32bit: 
|--offset-|--value--|
|--32bit--|--32bit--|

reg_value = value

64bit:
|-offset0-|-value0--|-offset1-|-value1-|
|--32bit--|--32bit--|--32bit--|--32bit--|
if(offset1 == offset0 + 1)
{
    reg_value = (value1<<32) | value0
    dword +=4 
}
else 
{
    reg_value = value0
    dword +=2
}

Reply:
Yes that's the right understanding